dashboard/test/ui/features/share_buttons.feature

```diff
@@ -0,0 +1,23 @@
+@as_student
+  # need @as_student to log in in order to get to Game Lab since it's a droplet game
+
+Feature: Share Buttons
+  # Making How It Works button show up in Sprite Lab and not in Game Lab
+
+Scenario: How It Works Button appears for Sprite Lab share page
+  Given I am on "http://studio.code.org/projects/spritelab"
+  And I wait for the page to fully load
+  When I navigate to the shared version of my project
+  # up till here gets you to that share page with the mobile screen but after that it takes you to the homepage
+  # "I navigate to the share URL" takes you to that share pg with mobile screen
+  And I wait until element "#open-workspace" is visible
+  Then I see "#open-workspace"
+
+  Scenario: How It Works Button appears for Game Lab share page
```

Comment:
I think the scenario title is missing a negation - that the button does not appear.

Reply:
Thank you for letting me know! I got too excited that the test worked! I'll go ahead and fix that now.